### PR TITLE
SEO optimization, some front fix, connect team members api, new dark theme colors

### PR DESCRIPTION
**Pull request by enescomruk72**
This PR introduces a set of comprehensive updates aimed at improving our web application's overall performance, usability, and appearance. Below is a detailed breakdown of the changes included:

- [ feat ]SEO Optimization: Implemented several SEO enhancements to improve our website's visibility and search engine ranking. This includes optimizing meta tags, improving loading times, and ensuring all content is crawlable by search engines.

- [ fix ] Front-end Fixes: Addressed various UI issues to enhance user experience. These fixes include resolving layout inconsistencies, improving responsive design for mobile devices, and refining interactive elements for better usability.

- [ feat ] Team Members API Integration: Connected our application with the Team Members API, enabling dynamic display of team member profiles. This integration allows for real-time updates and management of team member information directly through our admin panel.

- [ fix ] New Dark Theme Colors: Introduced a new color palette for our dark theme option. The updated colors are designed to improve readability and visual comfort for users preferring a darker interface.

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -1,14 +1,33 @@
...
+  <meta name="msapplication-TileColor" content="#2b5797" />
+  <meta name="theme-color" content="#000000" />
+</head>
+
+<body>
+  <div id="root"></div>
+  <script type="module" src="/src/main.jsx"></script>
+</body>
+
+</html>
```
Comment (CoderMungan) [inline frontend/index.html:33]: Gj

File: frontend/public/robots.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+User-agent: *
+Disallow:
+Sitemap: https://rythmtechnologies.com/sitemap.xml
```
Comment (CoderMungan) [inline frontend/public/robots.txt:3]: We must the have!

File: frontend/public/site.webmanifest [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        {
+            "src": "/android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#000000",
+    "background_color": "#000000",
+    "display": "standalone"
+}
```
Comment (CoderMungan) [inline frontend/public/site.webmanifest:24]: Best!

File: frontend/public/sitemap.xml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+	<loc>https://rythmtechnologies.com/rythm-akademi</loc>
+	<lastmod>2024-02-10T00:04:59+01:00</lastmod>
+	<priority>1.0</priority>
+</url>
+<url>
+	<loc>https://rythmtechnologies.com/projeler</loc>
+	<lastmod>2024-02-10T00:04:59+01:00</lastmod>
+	<priority>1.0</priority>
+</url>
+</urlset>
```
Comment (CoderMungan) [inline frontend/public/sitemap.xml:34]: Sitemap done good job !!

**Review by CoderMungan — APPROVED** [review]
LGTM! Great job!

File: frontend/public/robots.txt (hunk 2 above)
Comment (enescomruk72) [reply frontend/public/robots.txt:3]: Exactly :)

File: frontend/index.html (hunk 1 above)
Comment (enescomruk72) [reply frontend/index.html:33]: thx

File: frontend/public/site.webmanifest (hunk 3 above)
Comment (enescomruk72) [reply frontend/public/site.webmanifest:24]: This site.webmanifest will be updated soon.

File: frontend/public/sitemap.xml (hunk 4 above)
Comment (enescomruk72) [reply frontend/public/sitemap.xml:34]: Thx

**Pull request merged**